Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 139,
```

Comment:
Do you want to make the minor version 140 since we are versioning it sprint based.

Reply:
Done